### ISSUE TITLE
Make 'AS' optional in column and table alias

### ISSUE THREAD

**Issue by chrisknoll** [duckdb/duckdb#3449]
#### What happens?
If you try to alias a subquery such as:
```
select t.* from (
  select col1, col2, col3
  from table1
) t
```

You will get a syntax error because the FROM clause in the outer query should have used `AS t` when aliasing.

#### To Reproduce
Run the above query, see the error.

#### Environment (please complete the following information):
 - OS: [Windows]
 - DuckDB Version: [0.3.2-2]
 - DuckDB Client: [R]

#### Other notes:

I couldn't find any other RDBMS that required the AS keyword when aliasing (tho they all supported the use of AS as an optional thing).  It's a little inconsistent that when you alias a column or table, you require AS but on joins you do not...maybe that's why the other major RDBMS implementations made AS optional...don't know, but looking at our existing queries, it will be a bit of a nightmare to have to go to every query for every column and subquery and add an AS statement...so if you could make the AS optional that would be awesome.

**Comment by Mytherin** [comment]
This seems to work fine for me:

```sql
create table table1 as select 1 col1, 2 col2, 3 col3;
select t.* from (
  select col1, col2, col3
  from table1
) t;
-- ┌──────┬──────┬──────┐
-- │ col1 │ col2 │ col3 │
-- ├──────┼──────┼──────┤
-- │ 1    │ 2    │ 3    │
-- └──────┴──────┴──────┘
```

Could you try to make a reproducible example?

**Comment by chrisknoll** [comment]
Apologies, my mistake.  I received an error message about a parse error around a table alias (ex: from (subquery) tar1) and the error said 'error around tar1' and I checked the syntax at the [documentation ](https://duckdb.org/docs/sql/statements/select)and saw that it appeared AS was required.  I jumped the gun to write a issue too soon, and I'm sorry to waste your time.  The syntax works without AS.
